packages/remix/package.json

```diff
@@ -20,7 +20,7 @@
     "defaults"
   ],
   "dependencies": {
-    "@remix-run/dev": "npm:@vercel/remix-run-dev@1.14.0",
```

Comment:
Does this mean if we're patching our own `@remix-run/dev` on our project, that it won't take any effect because we're forced to use this version?